src/dmd/link.d

```diff
@@ -1046,7 +1046,7 @@ version (Windows)
             if (WindowsSdkDir is null)
             {
                 WindowsSdkDir = GetRegistryString(r"Microsoft\Windows Kits\Installed Roots", "KitsRoot10");
-                if (WindowsSdkDir && !FileName.exists(FileName.combine(WindowsSdkDir, "Lib")))
+                if (WindowsSdkDir && !findLatestSDKDir(FileName.combine(WindowsSdkDir, "Include"), r"um\windows.h"))
```

Comment:
All the hipsters use backticks nowadays ;-)

(FWIW the two alternative ways of doing WYSIWYG strings seem to be yet another historical baggage of D)

Reply:
Also the same logic is used here too:

https://github.com/dlang/dmd/blob/38355439498d9e7606bb590e92d3bf4bec372b25/src/dmd/link.d#L1080-L1084

Might make sense to avoid duplications as this code seems to be highly susceptible to failures.

Reply:
WindowsSdkVersion is no longer used after https://github.com/dlang/dmd/pull/7828, so this copy can be removed later.